tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -1,735 +1,176 @@
-                        Text="{x:Bind ViewModel.PageHeaderText}"/>
-                    <TextBlock
-                        Style="{ThemeResource SubtitleTextBlockStyle}"
-                        Visibility="{x:Bind ViewModel.CompletedWithErrors, Mode=OneWay}"
-                        x:Uid="ms-resource:///DevHome.SetupFlow/Resources/SummaryPage_HeaderWithErrors"/>
+        VerticalScrollMode="Enabled">
+        <Grid
+            MaxWidth="{ThemeResource MaxPageContentWidth}"
+            Margin="{ThemeResource ContentPageMargin}"
+            ColumnSpacing="{x:Bind ViewModel.ShowConfigurationFileResults, Mode=OneWay, Converter={StaticResource ConfigurationFileFlowColumnSpacing}}">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="auto" />
+                <RowDefinition />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="{x:Bind ViewModel.ShowConfigurationFileResults, Mode=OneWay, Converter={StaticResource ConfigurationFlowLeftColumnWidth}}" />
+                <ColumnDefinition Width="{x:Bind ViewModel.ShowConfigurationFileResults, Mode=OneWay, Converter={StaticResource ConfigurationFlowRightColumnWidth}}" />
+            </Grid.ColumnDefinitions>
+
+            <StackPanel
+                x:Name="SummaryHeader"
+                Grid.Row="0"
+                Grid.ColumnSpan="2"
+                Margin="0,20,0,40">
+                <!--  Display if the user has tasks need the machine to restart.  -->
+                <summaryViews:SummaryNeedsRestart
+                    Height="55"
+                    Padding="10"
+                    Background="{ThemeResource ComboBoxItemBackgroundPressed}"
+                    CornerRadius="5"
+                    DataContext="{x:Bind ViewModel}"
+                    Visibility="{x:Bind ViewModel.ShowRestartNeeded}" />
+
+                <!--  Introduction to the summary page via configuration results  -->
+                <summaryViews:SummaryIntroViaConfiguration DataContext="{x:Bind ViewModel}" Visibility="{x:Bind ViewModel.ShowConfigurationUnitResults, Mode=OneWay}" />
+
+                <!--  Introduction to the summary page via non-configuration flow  -->
+                <summaryViews:SummaryIntroViaNonConfigurationFlow DataContext="{x:Bind ViewModel}" Visibility="{x:Bind ViewModel.ShowConfigurationUnitResults, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}" />
+            </StackPanel>
+
+            <!--  Left Side  -->
+            <StackPanel Grid.Row="1" Grid.Column="0">
+
+                <!--  Configuration file flow.  Show stats.  -->
+                <Grid Visibility="{x:Bind ViewModel.ShowConfigurationFileResults, Mode=OneWay}">
+                    <Border BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}" BorderThickness="{ThemeResource TopLeftQuadrantThickness}">

```

Comment:
It would probably be simpler to set the BorderBrush and BorderThickness on the new summaryViews directly, rather than as separate containing elements.

Reply:
Yeah, I can see your point.  I wanted to keep each separate view SummaryView agnostic.  This makes SummaryView responsible for the UI look and feel and the other views should be concerned with their section.

Also, if the sections will move, the border should not move with them.